test/fixtures/kube-config/invalid_config.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 clusters:
 - cluster:
     insecure-skip-tls-verify: true
-    server: https://192.168.0.1
+    server: https://127.0.0.2
```

Comment:
Actually, this results in `The connection to the server 127.0.0.2 was refused - did you specify the right host or port?` on CI (connection refused instead of timeout). I could change the test assertion to look for that error, but it needs to happen consistently on dev/ci. What do you recommend @stefanmb ?

Reply:
Interesting, I tried locally with the 127.* addresses and I don't get a different error. I wonder if this is related to the Minikube setup on CI. We can leave as-is unless anyone reports breakage with their (test) setup.